cql2pgjson/src/main/java/org/folio/cql2pgjson/util/Cql2SqlUtil.java

```diff
@@ -97,67 +87,113 @@ public static String cql2string(String s, boolean quotedString) {
     return like.toString();
   }
 
+  /**
+   * Convert a CQL string to an SQL string. and SQL string constant escapes only
+   * the single quote ' by doubling it.
+   *
+   * @param s CQL string without leading or trailing double quote
+   * @return SQL string without leading or trailing single quote
+   */
+  public static String cql2string(String s) throws QueryValidationException {
+    StringBuilder like = new StringBuilder();
+    boolean backslash = false; // previous character is backslash
```

Comment:
Can you remove the misleading comment and add the same javadoc used in the other methods?

    /**
     * true if the previous character is an escaping backslash
     */